app/__init__.py

```diff
@@ -1,7 +1,37 @@
 from flask import Flask
+from flask_sqlalchemy import SQLAlchemy
+from flask_migrate import Migrate
+from dotenv import load_dotenv
+import os
+
+db = SQLAlchemy()
+migrate = Migrate()
+load_dotenv()
 
 
 def create_app(test_config=None):
     app = Flask(__name__)
 
+    # database = solar_system_development
```

Comment:
I think this is a comment but not commented out code. Is that right? If so, it might be more fitting to write this out in a sentence without using characters that can be interpreted as code because if this was accidentally uncommented out, then this would execute like code